src/aggregator/services/budget-insight/budget-insight.client.spec.ts

```diff
@@ -113,6 +114,35 @@ describe('BudgetInsightClient', () => {
     });
   });
 
+  it('returns the user connection information', async () => {
+    const connectionsResponse: ConnectionWrapper = {
+      connections: [
+        {
+          id: 4,
+          id_user: 6,
+          id_connector: 5,
+          last_update: null,
+          state: null,
+          active: true,
+          created: null,
+          next_try: null,
+        },
+      ],
+    };
+    result.data = connectionsResponse;
+    const token = 'token';
+    const spy = jest.spyOn(httpService, 'get').mockImplementationOnce(() => of(result));
+
+    const connections = await service.getConnectionInfo(token, '1');
+    expect(connections).toEqual(connectionsResponse);
+    expect(spy).toHaveBeenCalledWith('http://localhost:4000//users/me/connections/1/informations', {
```

Comment:
Tiny point, needs to be changed also in the config
```suggestion
    expect(spy).toHaveBeenCalledWith('http://localhost:4000/users/me/connections/1/informations', {
```

Reply:
This will change every endpoint in the tests. I'd rather change it in a separate PR.

Reply:
No problem ;)